6tempcontrol

```diff
@@ -297,22 +301,27 @@ do
     fi
 
     POWERLIMIT=${PWRLIMIT%%.*}
-    TEMP_DIFF=$(( TARGET_TEMP[GPU] - CURRENT_TEMP ))
+    TEMP_DIFF=$(( CURRENT_TEMP - TARGET_TEMP[GPU] ))
     NEW_FAN_SPEED=$CURRENT_FAN
     HIGH_TEMP_DIFF=$(( ALLOWED_TEMP_DIFF * 2 ))
 
     echo -e "${B}GPU $GPU${N}, Target temp: ${B}${TARGET_TEMP[${GPU}]}${N}, Current: ${B}$CURRENT_TEMP${N}, Diff: ${B}$TEMP_DIFF${N}, Fan: ${B}$CURRENT_FAN${N}, Power: ${B}$POWERDRAW${N}"
 
-    if (( CURRENT_TEMP > TARGET_TEMP[GPU] ))
+    if (( TEMP_DIFF > ALLOWED_TEMP_DIFF ))
```

Comment:
This should be reverted back to the way it was, so that we can fix on target and not letting it go above.